src/components/TruliooForm.jsx

```
@@ -6,8 +6,9 @@ import { getName } from "country-list"
 /** @jsx jsx */
 import { jsx, css } from '@emotion/core'
 
+let reservedFormDataKeys = new Set(["countries", "TruliooFields"])
```

I think you can move that part to the actions as well and let the action throw the exception.